srcpkgs/operator-sdk/template

```diff
@@ -1,7 +1,8 @@
 # Template file for 'operator-sdk'
 pkgname=operator-sdk
-version=0.18.1
-revision=3
+version=1.32.0
+revision=1
+archs="aarch64* x86_64*"
```

Comment:
Why the `archs` restriction?

Reply:
because upstream project only supports these 2 and ppc64le and s390x. when you then setup a project with operator-sdk, its going to go fetch some extra binaries from github that are also just supplied for those 4 archs.

here it fetches `kustomize` if not present on the `$PATH`: https://github.com/operator-framework/operator-sdk/blob/master/testdata/helm/memcached-operator/Makefile#L137

kustomize only supplies those 4 archs, and void-packages has no kustomize in repos, as a  fallback.
https://github.com/kubernetes-sigs/kustomize/releases/tag/kustomize%2Fv4.5.7

Reply:
are these binaries musl-compatible? if not, the `*` should not be there